### DISCUSSION TITLE
Do Site Resources Generally Go In An Assets Container?

### DISCUSSION THREAD

**Discussion by grovberg** [statamic/cms#8871] [General]
Short Version:
Are you guys creating an assets container for your site-wide resources (logos, backgrounds, icons) to link to from templates?

Long Version:
With the last CMS I worked with (this was MANY years ago), the equivalent to Assets was generally used only by content authors for article content. Resources for the site itself would go elsewhere in a regular directory, inaccessible from the control panel UI. I started building my first real Statamic site assuming that arrangement but keep finding little pain points (such as Vite not picking them up). I don't actually feel all that strongly about it either way, but I'm curious if there are community norms for a thing like this.

**Comment by jasonvarga** [comment]
Nope. You should only put content-managed assets in your asset containers.

Depending on your build process, site assets should either go in `resources` (if using a tool like Vite) or somewhere like `public/img` if you reference them directly.

If Vite isn't picking up your site assets maybe you need to configure it to look in the right place? Or make sure you are using the [Vite tag](https://statamic.dev/tags/vite#processing-static-assets-with-vite).
